packages/shared/src/components/cards/FeedbackCard.tsx

```diff
@@ -25,8 +25,8 @@ export const FeedbackCard = ({
 
   return (
     <div className="flex-1 space-y-4 p-6 pb-5">
-      <div className="relative flex justify-between">
-        <p className="font-bold typo-callout">
+      <div className="relative block">
+        <p className="font-bold typo-callout mr-0 tablet:mr-4">
```

Comment:
You mentioned that the range for the issue is at `656px to 690px` - should we be removing the margin for sizes bigger than `tablet`?

Reply:
Figured out it happens also on some bigger px but small ranges, so since it doesn't ruin anything left from tablet and above